js/mobile-menu.js

```diff
@@ -1,29 +1,29 @@
 document.querySelector('.menu-bars').addEventListener('click', () => {
```

Comment:
- [ ] The Navbar opens on mobile 👍🏽 . But, I the **X** / **close** Icon is missing. As per [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md),  I recommend we improve on this one as it is part of the requirements🥂. And oh - here are some screenshots for your eased refference

<table>
 </tbody>
 <tr>
 <td>
As you can see, it seems like we're missing an <b>X</b> button/icon to help us close the navbar<br/>
<img src="https://user-images.githubusercontent.com/35315311/175296368-62ae4ffa-c3b1-4324-a92f-dc55c75ce142.png" />
 </td>
 <td>
 Looks like we need a close button as shown here. It will improve your <a href="https://forums.mobirise.com/discussion/12188/when-mobile-responsive-menu-is-opened-need-x-where-hamburger-icon-is#:~:text=It%20makes%20it%20more%20user,the%20screen%20to%20close%20it).">users experience</a> while they use your awesome app/website<br>
 <img src="https://user-images.githubusercontent.com/35315311/175298062-5de83c43-ad71-4c75-aafc-880eb9665b6e.png"/>

 </td>
 </tr>
 </tbody>
</table>